pwnagotchi/ui/hw/libs/waveshare/oledhat/config.py

```diff
@@ -9,11 +9,11 @@
 # * | Info        :
 # ******************************************************************************/
 # Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documnetation files (the "Software"), to deal
+# of this software and associated documentation files (the "Software"), to deal
```

Comment:
This might be a third party module. It also looks like a license agreement or similar (it's not a good idea to have typos in legal text, but it's often a PITA to get them corrected because `raisins` -- err `reasons`). I tend to get asked to drop third party modules. If you can point out the upstream, I'll redirect the PR to them.